docs/source/requirements.rst

```diff
@@ -30,7 +30,12 @@ Google Cloud Platform (GCP)
 ---------------------------
 
 * `Google Cloud Command Line Interface <https://cloud.google.com/cli>`_
-* `kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl>`_: see `this link <https://cloud.google.com/kubernetes-engine/docs/release-notes>`_ for supported versions.
+* `kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl>`_  Make sure to install a version supported by ElasticBLAST:
+
+  * You must use a a kubectl version 1.25 or lower (1.26 is not supported!).  
```

Comment:
There was an extra "a" prior to kubectl. Also, the phrase in parenthesis is redundant unless you want to point out that the default binary download is 1.26.
```suggestion
  * You must use a kubectl version 1.25 or lower.  
```